tests/nataliia_romanchuk/homework_6/test_unpack.py

```diff
@@ -0,0 +1,10 @@
+from nataliia_romanchuk.homework6.unpack import unpack_recursive, unpack_while_loop
+import pytest
+
+@pytest.mark.parametrize("some_item", [[123, ["234", None], [[1], [23], [[123], 123, ["sdf", True]]]]])
+def test_unpack_while_loop(some_item):
+    assert unpack_while_loop(some_item) == [123, "234", None, 1, 23, 123, 123, "sdf", True]
+
+@pytest.mark.parametrize("some_item", [[123, ["234", None], [[1], [23], [[123], 123, ["sdf", True]]]]])
+def test_unpack_recursive(some_item):
+    assert unpack_recursive(some_item) == [123, "234", None, 1, 23, 123, 123, "sdf", True]
```

Comment:
no need in parametrize when only 1 case

you can dus such parametrize:

```
@pytest.mark.parametrize("func", [unpack_while_loop, unpack_recursive])
def test_unpack(func):
    some_item = [123, ["234", None], [[1], [23], [[123], 123, ["sdf", True]]]]
    assert func(some_item) == [123, "234", None, 1, 23, 123, 123, "sdf", True]

```

Reply:
fixed